packages/react/src/components/lozenge/lozenge.tsx

```diff
@@ -6,82 +6,36 @@ import { Icon, IconName } from '../icon/icon';
 
 const MAXIMUM_LENGTH = '312px';
 
-export type LozengeType = 'default' | 'success' | 'alert' | 'warning' | 'info' | 'disabled';
+export type LozengeVariant = 'neutral' | 'success' | 'alert' | 'warning' | 'info' | 'discovery';
 
 interface StyledLozengeProps {
     $isMobile: boolean;
-    $type?: LozengeType;
+    $isSubtle?: boolean;
+    $variant: LozengeVariant;
     theme: ResolvedTheme;
 }
 
-function getLozengeBackgroundColor({ $type, theme }: StyledLozengeProps): string {
-    switch ($type) {
-        case 'success':
-            return theme.notifications['success-1.2'];
-        case 'disabled':
-            return theme.greys['light-grey'];
-        case 'alert':
-            return theme.notifications['alert-2.2'];
-        case 'warning':
-            return theme.notifications['warning-3.2'];
-        case 'info':
-            // TODO: add this color in default themes
-            return '#f9f7fb';
-        case 'default':
-        default:
-            return theme.greys['light-grey'];
-    }
-}
-
-function getLozengeBorderColor({ $type, theme }: StyledLozengeProps): string {
-    switch ($type) {
-        case 'success':
-            return theme.notifications['success-1.1'];
-        case 'disabled':
-            return theme.greys['mid-grey'];
-        case 'alert':
-            return theme.notifications['alert-2.1'];
-        case 'warning':
-            return theme.notifications['warning-3.1'];
-        case 'info':
-            return theme.notifications['info-1.1'];
-        case 'default':
-        default:
-            return theme.greys['dark-grey'];
-    }
-}
-
-function getLozengeColor({ $type, theme }: StyledLozengeProps): string {
-    switch ($type) {
-        case 'success':
-            return theme.notifications['success-1.1'];
-        case 'disabled':
-            return theme.greys['mid-grey'];
-        case 'alert':
-            return theme.notifications['alert-2.1'];
-        case 'warning':
-            return theme.notifications['warning-3.1'];
-        case 'info':
-            return theme.notifications['info-1.1'];
-        case 'default':
-        default:
-            return theme.greys['dark-grey'];
-    }
+function getLozengeColor(
+    { $variant, $isSubtle, theme }: StyledLozengeProps,
+    propertyType: 'background' | 'border' | 'text',
+): string {
+    const subtleSuffix = $isSubtle ? '-subtle' : '';
+    return theme.component[`lozenge${subtleSuffix}-${$variant}-${propertyType}-color`];
```

Comment:
Je pense que j'inverserais le variant et subtle, puisque le subtle s'applique sur le variant.
```suggestion
    return theme.component[`lozenge-${$variant}${subtleSuffix}-${propertyType}-color`];
```